<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen4</class>
 <widget class="QDialog" name="screen4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FRAIL2FIT</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>460</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many litres of water do you drink daily?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="days">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>44</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>460</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;What kind of physical activities do you enjoy?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2op1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>460</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Playing outdoor games</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2op2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>460</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Playing video games/binge watching</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="q2op3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>460</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Both</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>460</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Serif&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Get ready to start your fitness journey !!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="s4nxt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>700</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
